manifests/pipeline.yaml

```diff
@@ -117,7 +130,7 @@ objects:
             kind: DockerImage
             # for now we just follow :master, but we may start freezing if
             # things become too unstable
```

Comment:
does this comment need to be moved to the definition of `COREOS_ASSEMBLER_IMAGE` ?

Reply:
Thanks, done!